binding.gyp

```diff
@@ -0,0 +1,19 @@
+{
+  "targets": [
+    {
+      "target_name": "erlpack",
+      "sources": [ "js/erlpack.cc" ],
+      "include_dirs": [
+        "<!(node -e \"require('nan')\")",
+        "electron/vendor/brightray/vendor/download/libchromiumcontent/src"
```

Comment:
is this required? it is most certainly not going to resolve to a valid path almost everywhere.